src/assets/toolkit/styles/templates/_print.scss

```diff
@@ -74,11 +74,22 @@
 .hide-for-print {
   @media print {
     display: none;
-  }  
+  }
 }
 
 .show-for-print {
   @media print {
     display: block;
-  }  
+  }
+}
+
+@media print {
+  [ng\:cloak], [ng-cloak], [data-ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak, .ng-hide:not(.ng-hide-animate) {
+   display: block !important;
```

Comment:
no we can keep the manual stuff, since we have control over how that looks/works/etc, my point is that the `ng-text-truncate` module as it currently works doesn't give us access to any unique classes that we can target for print purposes. So because it requires us to add workarounds to "show some .ng-hide things" we _could_ make it more explicit and just add `.hide-for-print` and `.show-for-print` classes _inside_ the module (if we fork it).